notNeededPackages.json

```diff
@@ -4369,6 +4369,10 @@
             "libraryName": "node-emoji",
             "asOfVersion": "2.1.0"
         },
+        "node-hid": {
+            "libraryName": "node-hid",
+            "asOfVersion": "2.2.0"
```

Comment:
I'm not seeing types in the published package until 3.0.0; did you find it in 2.2.0 somewhere?

Reply:
Fixed.